Add Olake to the Hacktoberfest Swag List for 2025 (#1039)
crweiner/hacktoberfest-swag-list: docs/list.md

- **How to sign up**: Join via the ["Sign Up Here" button on the home page](https://modtoberfest.com/).
- **Notes**: Read all of the [Modtoberfest Rules](https://modtoberfest.com/rules) and the [FAQ](https://modtoberfest.com/faq) to learn more.

### O

#### Olake

> - **Swag:**  
>   Exclusive Jacket, Diaries, Pens, Stickers, Hacktoberfest goodies

Reviewer: ```suggestion
- **Swag:** Jacket, diaries, pens and stickers.
```

> - **Requirements:**  
>     - Contribute to UI or Core repos during Hacktoberfest 2025  
>     - PRs must be merged  
>     - Tech: TypeScript, JavaScript, Golang  
>     - Beginners: “good first issues”; Advanced: deeper exploration  
>     - Active contributors may get priority for future openings

Reviewer: ```suggestion
- **Requirements:**  
		- Contribute to open issues in olake-UI or olake-Core repos. 
		- Get your PR merged and join [Olake at Slack](https://olake.io/slack/) to get support.
```

> - **How to sign up:**  
>   Join [Olake Slack](https://olake.io/slack/) and pick issues from:  
>     - [UI Repo](https://github.com/datazip-inc/olake-ui/issues)  
>     - [Core Repo](https://github.com/datazip-inc/olake/issues)  
>   Submit PRs during Hacktoberfest

Reviewer: ```suggestion
- **How to sign up:**  No known sign up link.
```

- **Issues:**  

>   All open Hacktoberfest-tagged issues

Reviewer: ```suggestion
		- [Olake - UI](https://github.com/datazip-inc/olake-ui/issues)
    - [Olake - Core](https://github.com/datazip-inc/olake/issues)
```

> - **Notes:**  
>   [Original Announcement](https://twitter.com/_olake)

Reviewer: ```suggestion
- **Notes:**  Read the [pinned post](https://x.com/cappybaradeploy/status/1973027154041729331) for more details.
```
